package.json

```diff
@@ -45,12 +45,12 @@
     "cheerio": "1.0.0-rc.2",
     "co": "4.6.0",
     "dox": "0.3.1",
-    "eslint": "5.16.0",
+    "eslint": "6.8.0",
     "eslint-plugin-mocha-no-only": "1.1.0",
     "highlight.js": "9.1.0",
     "lodash.isequal": "4.5.0",
     "lodash.isequalwith": "4.4.0",
-    "marked": "0.6.2",
+    "marked": "0.8.0",
```

Comment:
This change is a bit risky for a simple devDependency - a breaking change could break our docs build.

Reply:
FWIW, I ran website.js and didn't see any errors - the docs were generated successfully.